### PR TITLE
Expose collaborators endpoint - PLT-6915

### PR DESCRIPTION
**Pull request by kylegmaxwell**
This function is going to be used in the C# sdk for the issue
tracking plugin, so I am adding it here as well for completeness.

### CODE REVIEW

File: flux-sdk-common/src/serializers/collaborators-serializer.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function serialize(entity) {
```
Comment (tdb-alcorn) [inline flux-sdk-common/src/serializers/collaborators-serializer.js]: This isn't really necessary, but if you're trying to be consistent with the other serializers then this function should serialize a single collaborator object, and you should make a serializeList function that serializes an array of collaborators.

Comment (kylegmaxwell) [inline flux-sdk-common/src/serializers/collaborators-serializer.js]: ok, done

**Comment by tdb-alcorn** [comment]
Do you still want to merge this?

**Comment by kylegmaxwell** [comment]
Sure, might as well

**Pull request merged**